SPECS/containerized-data-importer/cdi.spec

```
@@ -16,7 +16,7 @@
 #
 
 Summary:        Container native virtualization
-Name:           containerized-data-importer
+Name:           cdi
 Version:        1.51.0
 Release:        1%{?dist}
```

Since we have bumped the `%changelog` :)
```suggestion
Release:        2%{?dist}
```

---

> Since we have bumped the `%changelog` :)
Done!